app/helpers/reactjs_helper.rb

```diff
@@ -4,31 +4,4 @@ def mount_react_component(name, selector, data = [])
       "$(tfm.reactMounter.mount('#{name}', '#{selector}', #{data}));".html_safe
     end
   end
-
-  def webpacked_plugins_js_for(*plugin_names)
```

Comment:
These helpers were introduced [recently](https://github.com/theforeman/foreman/pull/4733) and are meant to be used by plugins - they make sure files from plugins are loaded in the page. I believe we want to keep them.